modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithBackgroundParagraphBehavior.php

```diff
@@ -120,6 +120,11 @@ public function preprocess(&$variables) {
     // Get plugin configuration and save in vars for twig to use.
     $config = $this->getSettings($paragraph);
 
+    // If this paragraph is full-width, add the full-width library.
+    if (isset($config['text_background_full_width'])) {
```

Comment:
I think it'd be better to check for the actual value here.
```suggestion
    if (isset($config['text_background_full_width']) && $config['text_background_full_width'] === 'full-width-background') {
```